### ISSUE TITLE
Create a invasive species page 

### ISSUE THREAD

**Issue by SanderDevisscher** [inbo/reporting-rshiny-grofwildjacht#139]
Is it possible to add a page to the app with graphs/maps related to invasive alien species based on the [TrIAS](https://github.com/trias-project) workflow. 

If this is possible I would like to meet with you, @timadriaens en @damianooldoni to discuss how, what and when.

**Comment by mvarewyck** [comment]
Sounds interesting!

**Comment by timadriaens** [comment]
Yes, enthusiasm and creativity! So here's what I had in mind:

- a simple webpage where users can explore the [unified checklist of alien species](https://github.com/trias-project/unified-checklist) and the [checklist indicators](https://github.com/trias-project/indicators) derived from them. This would entail a simple dashboarded version of the checklist with intergration of the indicator scripts. Some example of a query a user could explore:

the trend in all alien species introductions in Flanders, Wallonie, Brussels, all Belgium
the trend in alien species introductions for a specific taxon (kingdom, classis etc. level)
the trend in alien species introductions for a specific environment, native range 
the trend in alien species introductions for a specific pathway
the trend in alien species introductions for a specific degree of establishment

So basically, to implement query possibilities for all the breakdowns:

environment (marine, terrestrial, freshwater)
taxon (kingdom --> family)
native range
pathway (level 1 and level 2 CBD)
invasion stage (introduced, established etc.)
source checklist
distribution (country-level: Belgium)

- similarly, a webtool that allows on the fly [occurrence indicators](https://github.com/trias-project/indicators/tree/master/data/output/GAM_outputs) at species level. These are basically graphs with number of observations and occupancy (within and outside protected areas) and trends.

The great @damianooldoni has written functions that create the indicator graphs.

**Comment by SanderDevisscher** [comment]
> distribution (country-level: Belgium)

also Flanders, Wallonia and Brussels

**Comment by SanderDevisscher** [comment]
https://github.com/trias-project/indicators/blob/master/data/interim/data_input_checklist_indicators.tsv
dit is de grote tabel met alle informatie voor checklist indicators

**Comment by SanderDevisscher** [comment]
https://github.com/trias-project/indicators

**Comment by damianooldoni** [comment]
Misschien interessant om te weten. De tabel die Sander heeft vermeld bonven in https://github.com/inbo/reporting-rshiny-grofwildjacht/issues/139#issuecomment-601178822 is de output van deze pipeline:
https://trias-project.github.io/indicators/get_data_input_checklist_indicators.html.

**Comment by SanderDevisscher** [comment]
https://docs.google.com/spreadsheets/d/1LeXXbry2ArK2rngsmFjz_xErwE1KwQ8ujtvHNmTVA6E/edit#gid=808615322

**Comment by SanderDevisscher** [comment]
Split of into several other issues
